examples/api/lib/cupertino/context_menu/cupertino_context_menu.1.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Flutter code sample for [CupertinoContextMenu].
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ContextMenuApp());
+
+class ContextMenuApp extends StatelessWidget {
+  const ContextMenuApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      theme: CupertinoThemeData(brightness: Brightness.light),
+      home: ContextMenuExample(),
+    );
+  }
+}
+
+class ContextMenuExample extends StatelessWidget {
+  const ContextMenuExample({super.key});
+
+  Animation<Decoration> _boxDecorationAnimation(Animation<double> animation) {
+    return DecorationTween(
+      begin: BoxDecoration(
+          color: CupertinoColors.systemYellow,
+          boxShadow: const <BoxShadow>[],
+          borderRadius: BorderRadius.circular(20.0),
+      ),
+      end: BoxDecoration(
+        color: CupertinoColors.systemYellow,
+        boxShadow: CupertinoContextMenu.kEndBoxShadow,
+        borderRadius: BorderRadius.circular(20.0),
+      ),
+    ).animate(
+      CurvedAnimation(
+        parent: animation,
+        curve: Interval(
+          0.0,
+          CupertinoContextMenu.animationOpensAt,
+      ))
+    );
```

Comment:
Nit: Would something like this work in order to avoid reinstantiating the DecorationTween?

```dart
  final DecorationTween _tween = DecorationTween(
    begin: BoxDecoration(
      color: CupertinoColors.systemYellow,
      boxShadow: const <BoxShadow>[],
      borderRadius: BorderRadius.circular(20.0),
    ),
    end: BoxDecoration(
      color: CupertinoColors.systemYellow,
      boxShadow: CupertinoContextMenu.kEndBoxShadow,
      borderRadius: BorderRadius.circular(20.0),
    ),
  );

  // Or just do this inline in the builder below?
  static Animation<Decoration> _boxDecorationAnimation(Animation<double> animation) {
    return _tween.animate(
      CurvedAnimation(
        parent: animation,
        curve: Interval(
          0.0,
          CupertinoContextMenu.animationOpensAt,
        ),
      ),
    );
  }
```

Reply:
This works as is with the `_tween` moved outside of the class to the top of the file under the imports. It looks like some variables get declared up there in some examples. Is this ok here too?

Reply:
I think either way would work, declaring _tween inside or outside of the class.  Maybe if it's inside the class it should be static?

Sorry I think I'm being overly pedantic since this is just an example, it's probably not a big deal even keeping it in the builder like it was originally.  In reality it would probably depend on the app it was used in to decide where to put it.